tests/lib/cla_check.py

```diff
@@ -24,7 +24,7 @@
 
 
 def get_emails_for_range(r):
-    output = check_output(["git", "shortlog", "-se", r])
+    output = check_output(["git", "shortlog", "-se", r]).decode("utf-8")
```

Comment:
(iirc) a common pattern I saw to make the py2/py3 compatible for this was to use `check_output(["git", "shortlog", "-se", r], universal_newlines=True)` but it's been a while since I dove deep into this, maybe someone else remembers this better.